nsiqcppstyle_exe.py

```diff
@@ -425,7 +425,12 @@ def GetActiveFilter(self):
     def GetFilterFile(self, filterfile):
         if not os.path.exists(filterfile):
             return None
-        f = open(filterfile, 'r')
+        try:
+            f = open(filterfile, 'r')
```

Comment:
Opening a file wouldn't result in DecodeError. This error would be faced by the reader, not the `open` function

Reply:
Agreed, reverted the code change.

Done.